Entities/src/main/java/tw/waterball/judgegirl/entities/exam/Exam.java

```diff
@@ -40,6 +40,9 @@
     @Valid
     private List<Question> questions = new ArrayList<>();
 
+    @Valid
+    private List<ExamParticipation> examParticipations = new ArrayList<>();
```

Comment:
please rename `ExamParticipation` to `Examinee`

don't forget the view model and the data model